CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+## [3.0.1] - 2018-01-21
+### Changed
+- Bumped Rubocop to 0.52.1 in response to CVE-2017-8418 (@nicoleheejin)
```

Comment:
To keep things consistent with the other PR's we should bump it to `~> 0.51.0` which satisfies the requirement to patch the vulnerability and reduces the number of new cops to satisfy.